Network/dedicated-load-balancer-license-management.md

```diff
@@ -24,7 +24,7 @@ CenturyLink Cloud customers with a dedicated Citrix Netscaler Load Balancer
 
 The Netscaler Load Balancer uses FlexLM licensing to license features on the product. A license check is done when the VPX boots. The licensing daemon (service) will start, check for a valid license and then stop the daemon (service). The VPX will then enable certain features determined by the license daemon (service). If a license is expired or invalid, the Netscaler will stop passing traffic.
 
-### How to Determine if a VPX is not Passing Traffic due to an Expired License
+### <a name="check-traffic">Determining If a VPX Is Not Passing Traffic Due to an Expired License</a>
```

Comment:
we should not be putting HTML into a markdown coded kb.  don't mix them as the flag contentIsHTML": false is set for this kb